docs/blog/pydantic-v2.md

```diff
@@ -0,0 +1,221 @@
+* What will be removed
+* How I'm intending to go about completing it and getting it released
+* Some idea of timeframe :fearful:
+
+Here goes...
+
+# Plan & Timeframe
+
+I'm currently taking a kind of "sabbatical" after leaving my last job to get pydantic V2 released.
+Why? Well I ask myself that quite often. I'm very proud of how much pydantic is used, but I'm less proud of its internals.
+Since it's something people seem to use quite a lot (26m downloads a month, Used 72k public repos on GitHub),
+I want it to be as good as possible.
+
+While I'm on the subject of why, how and my odd sabbatical: if you work for a large company who use pydantic a lot,
+you should encourage the company to **sponsor me a meaningful amount**, 
+like [Salesforce did](https://twitter.com/samuel_colvin/status/1501288247670063104).
+This is not charity, recruitment or marketing - the argument should be about how much the company will save if
+pydantic is 10x faster, more stable and more powerful - it would be worth paying me 10% of that to make it happen.
+
+The plan is to have pydantic V2 released within 3 months of full time work 
+(again, that'll be sooner if I can continue to work on it full time).
+
+Before pydantic V2 can be released, we need to released pydantic v1.10 - there are lots of changes in the main
+branch of pydantic contributed by the community, it's only fair to provide a release including those changes,
+many of them will remain unchanged for V2, the rest will act as a requirement to make sure pydantic V2 includes
+the capabilities they implemented.
+
+The basic road map for me is as follows:
+1. implement a few more critical features in pydantic-core
+2. release V0.1 of pydantic-core
+3. work on getting pydantic V1.10 out - basically merge all open PRs that are finished
+4. release pydantic V1.10
+5. delete all stale PRs which didn't make it into V1.10, apologise profusely to their authors who put their valuable
+  time into pydantic only to have their PRs closed :pray:
+6. change the main branch of pydantic to target v2
+7. start tearing pydantic code apart and see how many existing tests can be made to pass
+8. rinse, repeat
+9. release pydantic V2 :tada:
+
+# Introduction
+
+Pydantic began life as an experiment in some code in a long dead project.
+I ended up making the code into a package and releasing it.
+It got a bit of attention on hacker news when it was first released, but started to get really popular when
+Sebastian Ramirez used it in FastAPI.
+Since then the package and its usage have grown enormously.
+The core logic however has remained relatively unchanged since the initial experiment.
+It's old, it smells, it needs to be rebuilt.
+The release of version 2 is an opportunity to rebuild pydantic and correct many things that don't make sense.
+
+Much of the work on V2 is already done, but there's still a lot to do.
+Now seems a good opportunity to explain what V2 is going to look like and get feedback from uses.
+
+## Headlines
```

Comment:
I think this section could be clearer, it kind of bleeds into the next section of proposed changes and could be more punchy.

Reply:
rewritten.